label_sync/labels.yaml

```diff
@@ -1494,6 +1494,12 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: d2b48c
+        description: Issues with the Katacoda infrastructure for tutorials
+        name: team/katacoda
```

Comment:
Note AFAIK you also need to update prow config to allow people to /label team/katacoda

Reply:
It's likely plugins.yaml